apps/sms/views/conversation/js/recipients.js

```diff
@@ -20,7 +20,7 @@
     var number;
 
     opts = opts || {};
-    this.name = opts.name || opts.number || '';
+    this.name = String(opts.name || opts.number || '');
```

Comment:
ok; no I think it's good to make it sure we have a string here, let's keep this change.